### DISCUSSION TITLE
Test locally

### DISCUSSION THREAD

**Discussion by MatisseBE** [lennycolton/vatglasses-data#38] [Ideas]
Hi

Is there a way I can test the data I'm about to submit?

Kind regards

**Comment by lennycolton** [comment]
Hi Matisse,

I've added you to the testing repository. You'll need to upload a file called `belux.json`, which will then be visible at https://beta.vatglasses.uk/europe. I've removed the Maastricht placeholder sectors from the beta site.

Please be aware that it can take a couple of minutes for data to be deployed after you push it to the repository. If you push multiple times in quick succession, the server will process these requests one at a time - it can get very slow!

**Comment by MatisseBE** [comment]
Thank you! 

Just need help with one more thing...

Regex really arent my thing and not all of our COORD: are captured with the provided regex. 

Sometimes our degrees do not have a leading zero and/or if milliseconds are equal to zero, they are not written.
Could you take a look and perhaps update the wiki provided regex? 

![image](https://user-images.githubusercontent.com/51272243/213932822-2d7152c8-cb2b-47a1-9d94-0435acabe241.png)

**Reply by lennycolton** [reply]
Try this for the "find" with the same "replace": `COORD:N0{0,1}([0-9]{2})\.([0-9]{2})\.([0-9]{2})\.([0-9]{3}):E([0-9]{3})\.([0-9]{2})\.([0-9]{2})\.([0-9]{3})`

**Comment by MatisseBE** [comment]
Still doesnt work for those where milliseconds are removed such as:

COORD:N50.29.28:E004.42.01
COORD:N50.31.22:E004.50.25

Appologies for not providing an example for the second exception.

**Reply by lennycolton** [reply]
Aaargh your sector file is full of non-standard formatting. Try this: `COORD:N0?([0-9]{2})\.([0-9]{2})\.([0-9]{2})(\.[0-9]{3})?:E([0-9]{3})\.([0-9]{2})\.([0-9]{2})(\.[0-9]{3})?`

**Reply by MatisseBE** [reply]
Works brilliantly! Thank you!

**Reply by MatisseBE** [reply]
`COORD:N0?([0-9]{2}).([0-9]{2}).([0-9]{1,2})(.[0-9]{3,})?:E([0-9]{3}).([0-9]{2}).([0-9]{1,2})(.[0-9]{3,})?`

I made some more slight adjustments such that it could handle cases where the minutes only contained one digit.
